test/integration/apiserver/flowcontrol/concurrency_test.go

```diff
@@ -52,19 +52,19 @@ const (
 func setup(t testing.TB, maxReadonlyRequestsInFlight, MaxMutatingRequestsInFlight int) (*httptest.Server, *rest.Config, framework.CloseFunc) {
 	opts := framework.MasterConfigOptions{EtcdOptions: framework.DefaultEtcdOptions()}
 	opts.EtcdOptions.DefaultStorageMediaType = "application/vnd.kubernetes.protobuf"
-	masterConfig := framework.NewIntegrationTestControlPlaneConfigWithOptions(&opts)
+	controlPlaneConfig := framework.NewIntegrationTestControlPlaneConfigWithOptions(&opts)
 	resourceConfig := controlplane.DefaultAPIResourceConfigSource()
 	resourceConfig.EnableVersions(schema.GroupVersion{
 		Group:   "flowcontrol.apiserver.k8s.io",
 		Version: "v1alpha1",
 	})
-	masterConfig.GenericConfig.MaxRequestsInFlight = maxReadonlyRequestsInFlight
-	masterConfig.GenericConfig.MaxMutatingRequestsInFlight = MaxMutatingRequestsInFlight
-	masterConfig.GenericConfig.OpenAPIConfig = framework.DefaultOpenAPIConfig()
-	masterConfig.ExtraConfig.APIResourceConfigSource = resourceConfig
-	_, s, closeFn := framework.RunAnAPIServer(masterConfig)
+	controlPlaneConfig.GenericConfig.MaxRequestsInFlight = maxReadonlyRequestsInFlight
+	controlPlaneConfig.GenericConfig.MaxMutatingRequestsInFlight = MaxMutatingRequestsInFlight
+	controlPlaneConfig.GenericConfig.OpenAPIConfig = framework.DefaultOpenAPIConfig()
+	controlPlaneConfig.ExtraConfig.APIResourceConfigSource = resourceConfig
+	_, s, closeFn := framework.RunAnAPIServer(controlPlaneConfig)
```

Comment:
i think a problem around the master->controlPlane rename for test/integration was that this is not really a "control plane" config, but rather apiserver config.

is that true?

Reply:
I refer to the PR of comment on https://github.com/kubernetes/kubernetes/pull/100181#issuecomment-799591531

Reply:
https://github.com/kubernetes/kubernetes/pull/100181#discussion_r594825941 ? cc @pacoxu

Reply:
I see that the other names in that comment didn't happen though. and that after a bit of deliberation https://github.com/kubernetes/kubernetes/pull/100181#issuecomment-799591531 suggests to just use controlPlane here.